packages/editor-ui/src/components/RunDataTable.vue

```diff
@@ -12,7 +12,7 @@
 					@mouseenter="onMouseEnterCell"
 					@mouseleave="onMouseLeaveCell"
 				>
-					<n8n-text>{{ $locale.baseText('runData.emptyItemHint') }}</n8n-text>
+					<n8n-info-tip>{{ $locale.baseText('runData.emptyItemHint') }}</n8n-info-tip>
```

Comment:
is this centered still?

Reply:
No (I guess you mean vertically, the text compared to the icon), probably wouldn't work with multi-line text